.travis.yml

```diff
@@ -62,10 +62,10 @@ deploy:
     skip_cleanup: true
     on:
       repo: F5Networks/f5-openstack-agent
-      branch: mitaka
+      branch: stable/newton
       condition: $TRAVIS_TAG = ""
     script:
-    - ./docs/scripts/deploy-docs.sh publish-product-docs-to-prod openstack/agent $TRAVIS_BRANCH
+    - ./docs/scripts/deploy-docs.sh publish-product-docs-to-prod openstack/agent newton
```

Comment:
Is the branch name that get passed to deploy-docs script "newton" or "stable/newton"?